assets/package.json

```diff
@@ -11,15 +11,18 @@
     "test:watch": "vitest",
     "test:coverage": "vitest run --coverage",
     "test:ui": "vitest --ui",
-    "lint": "yarn lint:js && yarn lint:css",
+    "lint": "yarn prettier-check && yarn lint:js && yarn lint:css",
     "lint:js": "eslint ./src",
     "lint:css": "stylelint src/**/*.css",
-    "fix": "yarn fix:js && yarn fix:css",
+    "fix": "yarn prettier-write && yarn fix:js && yarn fix:css",
     "fix:js": "eslint ./src --fix",
     "fix:css": "stylelint src/**/*.css --fix",
     "e2e:start": "concurrently -kill-others --success first \"yarn start\" \"wait-on -c waitOnConfig.json -v https-get://localhost:3000 && cd e2e/ && npx cypress run\"",
     "e2e:open": "concurrently -kill-others --success first \"yarn start\" \"wait-on -c waitOnConfig.json -v https-get://localhost:3000 && cd e2e/ && npx cypress open\"",
-    "graphql:codegen": "graphql-codegen --config codegen.yml"
+    "graphql:codegen": "graphql-codegen --config codegen.yml",
+    "prettier-write": "yarn prettier --write ./src",
+    "prettier-check": "yarn prettier --check ./src",
+    "postinstall": "cd .. && husky install assets/.husky"
```

Comment:
We can run it automatically like [here](https://github.com/kubernetes/dashboard/blob/master/modules/web/.yarnrc.yml#L9) with [postinstall](https://github.com/kubernetes/dashboard/blob/master/modules/web/.yarn/plugins/%40yarnpkg/plugin-postinstall.cjs) plugin.

Reply:
Oh great! I was struggling to find a way to get husky installed automatically with yarn. I'll take a look at this plugin.